salt/edx/templates/ansible_env_config.yml.j2

```diff
@@ -238,37 +294,44 @@ lms_env_config:  &lms_env
       KEY_PREFIX: "integration_static_files"
       LOCATION:
         - "localhost:11211"
-  CELERY_BROKER_HOSTNAME: "rabbitmq.service.consul"
-  CELERY_BROKER_TRANSPORT: "amqp"
-  CELERY_BROKER_VHOST: '{{ rabbitmq_vhost }}'
-  CERT_QUEUE: "certificates"
-  CMS_BASE: "{{ CMS_NAME }}"
-  HOSTNAME_MODULESTORE_DEFAULT_MAPPINGS:
-    'preview\.': 'draft-preferred'
-  CODE_JAIL:
-    limits:
-      REALTIME: 3
-      CPU: 3
-      FSIZE: 1048576
-      PROXY: 0
-      VMEM: 536870912
-    python_bin: "{{ edxapp_sandbox_venv_dir }}/bin/python"
-    user: "{{ edxapp_sandbox_user }}"
-  COMMENTS_SERVICE_KEY: "{{ COMMENTS_SERVICE_KEY }}"
-  COMMENTS_SERVICE_URL: "http://localhost:4567"
-  DEFAULT_FEEDBACK_EMAIL: "{{ DEFAULT_FEEDBACK_EMAIL }}"
-  DEFAULT_FROM_EMAIL: "{{ DEFAULT_FROM_EMAIL }}"
-  EMAIL_BACKEND: "django.core.mail.backends.smtp.EmailBackend"
-  FEEDBACK_SUBMISSION_EMAIL: ""
+  SEGMENT_IO_LMS: true
+  SERVER_EMAIL: "{{ SERVER_EMAIL }}"
+  SESSION_COOKIE_DOMAIN: !!null
+  SESSION_ENGINE: "django.contrib.sessions.backends.db"
+  SYSLOG_SERVER: "{{ SYSLOG_SERVER }}"
+  TIME_ZONE_DISPLAYED_FOR_DEADLINES: "{{ TIME_ZONE }}"
+  TIME_ZONE: "{{ TIME_ZONE }}"
+  WIKI_ENABLED: true
+  STATIC_ROOT_BASE: "/edx/var/edxapp/staticfiles"
+  CAS_SERVER_URL: '{{ CAS_SERVER_URL }}'
+  CAS_EXTRA_LOGIN_PARAMS:
+    provider: '{{ CAS_LOGIN_PROVIDER }}'
+    appname: '{{ CAS_LOGIN_APPNAME }}'
+  CAS_ATTRIBUTE_CALLBACK:
+    module: '{{ CAS_ATTRIBUTE_CALLBACK_MODULE }}'
+    function: '{{ CAS_ATTRIBUTE_CALLBACK_FUNCTION }}'
+  ORA2_FILE_PREFIX: "{{ env }}-dev/ora2"
+  FILE_UPLOAD_STORAGE_PREFIX: "{{ UPLOAD_STORAGE_PREFIX }}"
+  MEDIA_URL: ""
+  FILE_UPLOAD_STORAGE_BUCKET_NAME: "{{ AWS_STORAGE_BUCKET_NAME }}"
+
+lms_env_config:
+  <<: *generic_env_config
+  COURSES_WITH_UNSAFE_CODE: "{{ COURSES_WITH_UNSAFE_CODE }}"
+  BULK_EMAIL_DEFAULT_FROM_EMAIL: "{{ BULK_EMAIL_DEFAULT_FROM_EMAIL }}"
+  PLATFORM_NAME: "{{ PLATFORM_NAME }}"
+  # THEME_NAME: "{{ THEME_NAME }}"
+  MKTG_URL_LINK_MAP:
+    CONTACT: !!null
+    FAQ: !!null
+    HONOR: !!null
+    PRIVACY: !!null
   GRADES_DOWNLOAD:
     'STORAGE_TYPE': 'S3'
     'BUCKET': "{{ AWS_GRADES_BUCKET_NAME }}"
     'ROOT_PATH': "{{ AWS_GRADES_ROOT_PATH }}"
-  LMS_BASE: "{{ LMS_NAME }}"
-  LOCAL_LOGLEVEL: "DEBUG"
   LOGGING_ENV: "{{ LMS_LOG_ENV }}"
   LOG_DIR: "{{ LMS_LOG_DIR }}"
-  MEDIA_URL: ""
   FEATURES:
```

Comment:
Nit: These feature flags are mostly have to be the same between LMS and CMS, so moving these out to `generic_env_config` or a separate hash merged in (if necessary) can trim more repetition.